src/workerd/io/compatibility-date.capnp

```diff
@@ -692,4 +692,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $experimental
       $neededByFl;
   # Enables cache settings specified request in fetch api cf object to override cache rules. (only for user owned or grey-clouded sites)
+
+  ctxPropsToDefault @73: Bool
+      $compatEnableFlag("ctx_props_to_default_enabled")
+      $compatDisableFlag("ctx_props_to_default_disabled")
+      $experimental;
```

Comment:
I think there's no need to mark this experimental, in fact let's go ahead and give it an enable date of `2025-03-03`.